tests/unit/io/test_fileio.py

```diff
@@ -89,14 +94,16 @@ def test_find_files_when_file_absent(tmp_path):
 
 @pytest.mark.parametrize("filesystem", REMOTE_FS_PREFIX)
 def test_is_remote_when_using_remote_prefix(filesystem):
-    """is_remote returns True when path starts with one of the TFX remote file prefixes"""
+    """is_remote returns True when path starts with one of
+    the TFX remote file prefixes"""
```

Comment:
The docstring might be a bit misleading here, as the check is based on the prefixes which reside in our `zenml.constants`

Reply:
Fixed.